scripts/configs/pg4.yaml

```diff
@@ -0,0 +1,53 @@
+# PG4 experiments.
+---
+APPROACHES:
+  pg3:
+    NAME: "pg3"
+    FLAGS: []
```

Comment:
note @wester-is-not-real : tom realized that all the "FLAGS: []" should actually be "FLAGS: {}" (see scripts/configs/example_basic.yaml)

Reply:
Thank you!